mongodb/src/main/scala/akka/stream/alpakka/mongodb/Mongo.scala

```diff
@@ -0,0 +1,16 @@
+package akka.stream.alpakka.mongodb
+
+import com.typesafe.config.{Config, ConfigFactory}
+import org.mongodb.scala.{MongoClient, MongoDatabase}
+
+object Mongo {
+  private val config: Config = ConfigFactory.load(getClass.getClassLoader, "application.conf")
```

Comment:
Would be better if there was a DSL for the connection details that is passed to the factory methods (possibly with an optional parse-from-config method but I'm not sure that is needed). The current solution will severly limit the usefulness of the connector since you could only ever connect to a single mongodb defined in `application.conf`. Look at how it is done in for example the amqp connector. Could be as simple as `case class MongoConnectionDetails(db, host, port)`.